share/revng/pipelines/revng-c-pipelines.yml

```diff
@@ -12,6 +12,8 @@ Containers:
     Type: decompiled-c-code
   - Name: decompiled.tar.gz
     Type: decompile
+  - Name: decompiled-directory.tar.gz
+    Type: decompiled-to-directory
```

Comment:
I'd go for `recompilable-archive.tar.gz` and `recompilable-archive`.
We talk about `archive`, not `directory`, let's strive for coherency.

Reply:
Done